test/features/build_server/cache/project_cache_test.dart

```diff
@@ -9,7 +16,72 @@ import '../../../helpers/create_dummy_project_cache.dart';
 const _cacheId = 'project_cache_test';
 const _dummyProjectCachePath = './flyde-test-lib-$_cacheId/cache/testing';
 
+/// Reads the `./example` directory and returns a list of all files.
```

Comment:
I would move these helpers to the bottom, main (what a pun) content first

Reply:
```suggestion
/// Reads the `./example` directory and returns a list of all cpp related files.
```

Reply:
I prefer to put helpers at the top but I don't care enough to not resolve this :)